azure-pipelines.yml

```diff
@@ -109,198 +109,3 @@ jobs:
         displayName: Cypress E2E tests
 
       - template: .devops/templates/cleanup.yml
-
```

Comment:
will not be deleted when merging this PR(but will in a follow up PR), it was deleted to test the functionality of the workflows